qa-deploy

```diff
@@ -78,14 +78,17 @@ function deploy_ingress() {
 }
 
 function run() {
+    production="${1:-}"
+    staging="${2:-}"
+    if [ -z "${production}" ]; then invalid "Missing production url"; fi
+
     echo -n "Are you connected to the VPN? [Y/n] "
     read vpn
-
     if [ "${vpn}" == "n" ]; then exit 0; fi
 
-    production="${1:-}"
-    staging="${2:-}"
-    if [ -z "${production}" ]; then invalid "Missing production url"; fi
+    echo -n "Have you pushed the image to prod-comms.docker-registry.canonical.com/${production}:latest? [Y/n] "
```

Comment:
Isn't there another way of checking if the image is pushed directly from the registry? Just to avoid to get 2 questions when we launch the script

Reply:
Yeah I'm gonna change this solution a bit...